app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    render template: 'layouts/users/index'
+  end
+
+  def show
+    render template: 'layouts/users/show'
+  end
+end
```

Comment:
- [ ] The same happens with your `users` views, move the folder outside of `layouts` and then you're good to remove the `render template:` instruction. 🤓 🔧 

It's okay if the action is empty for now, Rails will render the correct template which is the requirement for this milestone, so don't worry!